tests/test_data/test_compose.py

```diff
@@ -4,6 +4,15 @@
 
 from mmaction.datasets.pipelines import Compose, ImageToTensor
 
+try:
+    import torchvision
+    from distutils.version import LooseVersion
+    torchvision_ok = False
+    if LooseVersion(torchvision.__version__) >= LooseVersion('0.8.0'):
+        torchvision_ok = True
+except (ImportError, ModuleNotFoundError):
+    torchvision_ok = False
```

Comment:
No need to try-except

Reply:
Alternatives

```python
TORCHVISION_VERSION = tuple(int(x) for x in torchvision.__version__.split('.')[:3])

def obsolete_torchvision_version(torchvision_version, version_threshold):
    return torchvision_version<= version_threshold
```

Reply:
try except is needed since torchvision should not be added to known-third-parties